### PR TITLE
Added UIViewTests

### PR DESCRIPTION
**Pull request by lfarah**
More tests, more ❤️✅💪🚀

### CODE REVIEW

File: EZSwiftExtensionsTests/UIViewTests.swift [hunk 1]
```diff
@@ -7,10 +7,27 @@
...
+        let sub1 = UIView()
+        let sub2 = UIView()
+        let sub3 = UIView()
+        
+        view.addSubview(sub1)
+        view.addSubview(sub2)
+        view.addSubview(sub3)
+        
+        XCTAssertEqual(view.subviews.count, 3)
+        XCTAssertEqual(view.subviews.contains(sub1) , true)
```
Comment (Khalian) [inline EZSwiftExtensionsTests/UIViewTests.swift]: For the sake of completeness we should do the contains check for all subviews.

File: EZSwiftExtensionsTests/UIViewTests.swift [hunk 2]
```diff
@@ -7,10 +7,27 @@
...
+        
+        view.removeSubviews()
+        XCTAssertEqual(view.subviews.count, 0)
+        XCTAssertEqual(view.subviews.contains(sub1) , false)
```
Comment (Khalian) [inline EZSwiftExtensionsTests/UIViewTests.swift]: Same as above.

File: EZSwiftExtensionsTests/UIViewTests.swift (hunk 1 above)
Comment (piv199) [inline EZSwiftExtensionsTests/UIViewTests.swift]: Also we should check by position, you should check array equality

Comment (piv199) [inline EZSwiftExtensionsTests/UIViewTests.swift]: @lfarah `XCTAssertEqual(view.subviews, [sub1, sub2, sub3])` ?

**Comment by lfarah** [comment]
@piv199 done!

File: EZSwiftExtensionsTests/UIViewTests.swift [hunk 3]
```diff
@@ -7,10 +7,32 @@
...
+        let sub1 = UIView()
+        let sub2 = UIView()
+        let sub3 = UIView()
+        
+        view.addSubview(sub1)
+        view.addSubview(sub2)
+        view.addSubview(sub3)
+        
+        XCTAssertEqual(view.subviews, [sub1, sub2, sub3])
+        XCTAssertEqual(view.subviews.count, 3)
```
Comment (piv199) [inline EZSwiftExtensionsTests/UIViewTests.swift]: remove this

File: EZSwiftExtensionsTests/UIViewTests.swift [hunk 4]
```diff
@@ -7,10 +7,32 @@
...
+        XCTAssertEqual(view.subviews.contains(sub1) , true)
+        XCTAssertEqual(view.subviews.contains(sub2) , true)
+        XCTAssertEqual(view.subviews.contains(sub3) , true)
 
+        view.removeSubviews()
+        XCTAssertEqual(view.subviews.count, 0)
```
Comment (piv199) [inline EZSwiftExtensionsTests/UIViewTests.swift]: `XCTAssertEqual(view.subviews, [])`

File: EZSwiftExtensionsTests/UIViewTests.swift [hunk 5]
```diff
@@ -7,10 +7,32 @@
...
+        let sub2 = UIView()
+        let sub3 = UIView()
+        
+        view.addSubview(sub1)
+        view.addSubview(sub2)
+        view.addSubview(sub3)
+        
+        XCTAssertEqual(view.subviews, [sub1, sub2, sub3])
+        XCTAssertEqual(view.subviews.count, 3)
+        XCTAssertEqual(view.subviews.contains(sub1) , true)
```
Comment (piv199) [inline EZSwiftExtensionsTests/UIViewTests.swift]: remove this

File: EZSwiftExtensionsTests/UIViewTests.swift [hunk 6]
```diff
@@ -7,10 +7,32 @@
...
+        XCTAssertEqual(view.subviews.contains(sub2) , true)
```
Comment (piv199) [inline EZSwiftExtensionsTests/UIViewTests.swift]: remove this

File: EZSwiftExtensionsTests/UIViewTests.swift [hunk 7]
```diff
@@ -7,10 +7,32 @@
...
+        XCTAssertEqual(view.subviews.contains(sub3) , true)
```
Comment (piv199) [inline EZSwiftExtensionsTests/UIViewTests.swift]: remove this

File: EZSwiftExtensionsTests/UIViewTests.swift [hunk 8]
```diff
@@ -7,10 +7,32 @@
...
 
+        view.removeSubviews()
+        XCTAssertEqual(view.subviews.count, 0)
+        XCTAssertEqual(view.subviews.contains(sub1) , false)
```
Comment (piv199) [inline EZSwiftExtensionsTests/UIViewTests.swift]: remove this

File: EZSwiftExtensionsTests/UIViewTests.swift [hunk 9]
```diff
@@ -7,10 +7,32 @@
...
+        XCTAssertEqual(view.subviews.contains(sub2) , false)
```
Comment (piv199) [inline EZSwiftExtensionsTests/UIViewTests.swift]: remove this

File: EZSwiftExtensionsTests/UIViewTests.swift [hunk 10]
```diff
@@ -7,10 +7,32 @@
...
+        XCTAssertEqual(view.subviews.contains(sub3) , false)
```
Comment (piv199) [inline EZSwiftExtensionsTests/UIViewTests.swift]: remove this

**Comment by lfarah** [comment]
@piv199 what do you think?

**Pull request merged**